### PR TITLE
docs: fix variable name typo in run experiments doc

### PR DESCRIPTION
**Pull request by chrishart0**
The `ds` variable is never defined. This should have been `df` which is defined on line 32.

### CODE REVIEW

**Comment by chrishart0** [comment]
I have read the CLA Document and I hereby sign the CLA

**Comment by chrishart0** [comment]
recheck

File: docs/datasets-and-experiments/how-to-experiments/run-experiments.md [hunk 1]
```diff
@@ -176,9 +176,9 @@ Running an experiment is as easy as calling `run_experiment` with the components
 ```python
 from phoenix.experiments import run_experiment
 
-run_experiment(ds, task=task, evaluators=[no_error, has_results])
+run_experiment(df, task=task, evaluators=[no_error, has_results])
```
Comment (mikeldking) [inline docs/datasets-and-experiments/how-to-experiments/run-experiments.md:179]:
```suggestion
run_experiment(dataset, task=task, evaluators=[no_error, has_results])
```

**Comment by mikeldking** [comment]
Thanks for the catch @chrishart0 !!

**Pull request merged**